etc/helm/pachyderm/templates/cloudsqlAuthProxy/deployment.yaml

```diff
@@ -39,11 +39,20 @@ spec:
           - "-instances={{ .Values.cloudsqlAuthProxy.connectionName }}=tcp:0.0.0.0:{{ .Values.cloudsqlAuthProxy.port }}"
         ports:
         - containerPort: {{ .Values.cloudsqlAuthProxy.port }}
-        securityContext:
-          # The default Cloud SQL proxy image runs as the
-          # "nonroot" user and group (uid: 65532) by default.
-          runAsNonRoot: true
         resources: {{ toYaml .Values.cloudsqlAuthProxy.resources | nindent 10 }}
+      {{- if .Values.global.securityContexts.enabled }}
+        securityContext:
+          allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: true
+          capabilities:
+              drop:
+              - all
+      securityContext:
```

Comment:
Sorry, I might have missed this. `seccompProfile` needs to be on pod securitycontext.
```
        runAsUser: {{ .Values.global.securityContexts.runAsUser }}
        runAsGroup: {{ .Values.global.securityContexts.runAsGroup }}
        fsGroup: {{ .Values.global.securityContexts.runAsGroup }}
        runAsNonRoot: true
        seccompProfile:
          type: RuntimeDefault
```